package.json

```
@@ -36,6 +36,7 @@
     "luxon": "1.25.0",
     "nuxt": "2.14.6",
     "nuxt-fontawesome": "0.4.0",
+    "tzdata": "1.0.25",
```

exceptionally  use ~ in version so timezones will upgrade on deploy (last number, 25, 26, 27...)

```suggestion
    "tzdata": "~1.0.25",
```

---

DONE!